_posts/blog/2019-06-20-Hello!I am Yini He.md

```diff
@@ -1,10 +1,10 @@
 ---
```

Comment:
This is looking great @minnieho1115! Just a few comments and then we should be ok to go.

Please can you rename the file without the `!`? I think it will confuse some websites to have it in the filename.

Reply:
Thank Kirstie for giving me lots of great comments!